src/OrbitTriggerCaptureVulkanLayer/CMakeLists.txt

```diff
@@ -19,18 +16,16 @@ target_sources(OrbitTriggerCaptureVulkanLayer PRIVATE
 
 target_link_libraries(OrbitTriggerCaptureVulkanLayer PUBLIC
         CONAN_PKG::vulkan-headers
+        CONAN_PKG::protobuf
         OrbitBase
         OrbitCaptureGgpClientLib
-        OrbitTriggerCaptureVulkanLayerProtos
         Vulkan::ValidationLayers)
 
-project(OrbitTriggerCaptureVulkanLayerProtos)
-add_library(OrbitTriggerCaptureVulkanLayerProtos STATIC)
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/protos/OrbitTriggerCaptureVulkanLayer)
+protobuf_generate(TARGET OrbitTriggerCaptureVulkanLayer PROTOS
+        layer_config.proto
+        PROTOC_OUT_DIR
+        ${CMAKE_CURRENT_BINARY_DIR}/protos/OrbitTriggerCaptureVulkanLayer)
 
-protobuf_generate(TARGET OrbitTriggerCaptureVulkanLayerProtos PROTOS 
-        layer_config.proto)
-
-target_include_directories(OrbitTriggerCaptureVulkanLayerProtos PUBLIC 
-        ${CMAKE_CURRENT_BINARY_DIR})
-target_link_libraries(OrbitTriggerCaptureVulkanLayerProtos PUBLIC
-        CONAN_PKG::protobuf)
+target_include_directories(OrbitTriggerCaptureVulkanLayer PRIVATE
+        ${CMAKE_CURRENT_BINARY_DIR}/protos/OrbitTriggerCaptureVulkanLayer)
```

Comment:
So in this case the generated files are included without the module name because the inclusion is only from inside the module?

Reply:
Yes. That felt more natural at the point of writing. But I'm not sure anymore. I can change it if you like.

Reply:
I don't really have an opinion. On the one hand in the cases of the other protos we have the module name, on the other hand they are only used internally so they are consistent with headers internal to a module.

Reply:
I think I'm going to change it, just ot have one consistent way in the codebase, so if anyone copies that CMakeLists file for their new module they will use it correctly.

Reply:
> if anyone copies that CMakeLists file for their new module they will use it correctly

Good point!

Reply:
Done. Thanks for the review!